src/main/java/org/jabref/logic/importer/util/GrobidService.java

```diff
@@ -0,0 +1,53 @@
+package org.jabref.logic.importer.util;
+
+import java.io.IOException;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+
+import org.jabref.logic.net.URLDownload;
+
+/**
+ * Implements an API to a GROBID server, as described at
+ * https://grobid.readthedocs.io/en/latest/Grobid-service/#grobid-web-services
+ * <p>
+ * Note: Currently a custom GROBID server is used...
```

Comment:
I guess it would be a good idea to have the custom GROBID server also as a repository under the JabRef umbrella organization. What do you think? refs also @koppor

Reply:
Sure this makes sense. The Grobid server is here: https://github.com/kermitt2/grobid
The customized is here: https://github.com/NikodemKch/grobid
They are exactly the same except this commit: https://github.com/NikodemKch/grobid/commit/e89810bd82b1b39cfcf82150d37b2200ac510a30

Reply:
Once this merge request is accepted we can also use the default grobid server (Thank you @koppor ): https://github.com/kermitt2/grobid/pull/532/files

Reply:
I have added the header, but commented out with a TODO comment, since GROBID does not work with this header right now